### DISCUSSION TITLE
Can't find custom model, when running a query

### DISCUSSION THREAD

**Discussion by alanjohn13** [imartinez/privateGPT#436] [General]
When running query getting error
  ptj_model_load: loading model from 'models/ggml-vic13b-q5_1.bin' - please wait ...
  gptj_model_load: invalid model file 'models/ggml-vic13b-q5_1.bin' (bad magic)
  GPT-J ERROR: failed to load model from models/ggml-vic13b-q5_1.bin

Able to ingest data and is creating db folder.
I downloaded a new model from hugging face and updated .env file with that config, after placing the raw model file in models folder.

Env config
  PERSIST_DIRECTORY=db
  MODEL_TYPE=GPT4All
  MODEL_PATH=models/ggml-vic13b-q5_1.bin
  EMBEDDINGS_MODEL_NAME=all-MiniLM-L6-v2
  MODEL_N_CTX=100

 - OS / hardware: [e.g. macOS Ventura 13.3.1]
 - Python version [e.g. 3.10.3]
- Using custom model downloaded from hugging face placed in models fodler

**Comment by camiloteram** [comment]
Same deal

**Comment by pseudotensor** [comment]
h2GPT supports custom models just fine, as long as one knows the prompt_type associated with the model.  I recommend using wizardlm uncensored 7B or 13B on CPU using TheBloke (https://github.com/h2oai/h2ogpt#cpu) or on GPU directly (any HF model is supported, as long as prompt_type is correctly set)

**Comment by camiloteram** [comment]
I realized what the issue was: in the env config; the model type for the specific one is not GPT4ALL. 
It is LLaMA or something along those line. Worked fine afterwards.  

Env config

>>>> MODEL_TYPE=GPT4All <<<<<<
MODEL_PATH=models/ggml-vic13b-q5_1.bin
